### PR TITLE
Add 'New Order (OMIS) added' activity stream card

### PR DESCRIPTION
**Pull request by paulgain**
**Implementation**
OMIS

**Design**
https://drive.google.com/drive/folders/1UDKG1iOrGHISVyPF6z1tvGdllAZZj0gd

**Ticket**
https://trello.com/c/6i1U7NhA/1280-add-omis-order-activity-to-the-activity-timeline

### CODE REVIEW

**Comment by paulgain** [comment]
Teams are not going to be added to OMIS, please refer to the ticket for more information.

File: src/activity-feed/activity-feed-cards/Omis.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  Card,
+  CardContent,
+  CardDetails, CardHeading,
+  CardHeadingBlock,
+  CardMeta,
+  CardTable,
+} from './card'
+
+import CardUtils from './card/CardUtils'
+import {get} from "lodash";
```
Comment (paulgain) [inline src/activity-feed/activity-feed-cards/Omis.jsx]: I've just spotted the semicolon.

File: src/activity-feed/activity-feed-cards/Omis.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    const published = get(activity, 'published')
+    const reference = get(activity, 'object.name')
+    const country = get(activity, 'object.dit:country.name')
+    const ukRegion = get(activity, 'object.dit:ukRegion.name')
+    const url = get(activity, 'object.url')
+
+    return (
+      <Card>
+        <CardContent>
+          <CardHeadingBlock text={'New Order (OMIS) added'} />
```
Comment (yeahfro) [inline src/activity-feed/activity-feed-cards/Omis.jsx]: This line could be `<CardHeadingBlock text="New Order (OMIS) added" />`

File: src/activity-feed/activity-feed-cards/Omis.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          <CardHeading link={{ url, text: reference }}/>
+          <CardDetails
+            summary="View key details and people for this order"
+            link={{ url, text: 'Go to the order detail page' }}>
+            <CardTable rows={
+              [
+                { header: 'Country:', content: country },
```
Comment (yeahfro) [inline src/activity-feed/activity-feed-cards/Omis.jsx]: We haven't used colons in other places.

File: fixtures/activity_feed/index.js [hunk 4]
```diff
@@ -5,4 +5,5 @@ module.exports = [
   require('./investment_projects/project_added_fdi'),
   require('./investment_projects/project_added_non_fdi'),
   require('./investment_projects/project_added_cti'),
+  require('./omis/omis-add'),
```
Comment (yeahfro) [inline fixtures/activity_feed/index.js]: Would a better name for this be `omis_added` or `order_added`? Based upon the investment project fixtures

File: src/activity-feed/activity-feed-cards/Omis.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (paulgain) [reply src/activity-feed/activity-feed-cards/Omis.jsx]: Done

File: src/activity-feed/activity-feed-cards/Omis.jsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  CardDetails, CardHeading,
+  CardHeadingBlock,
+  CardMeta,
+  CardTable,
+} from './card'
+
+import CardUtils from './card/CardUtils'
+import {get} from "lodash";
+
+export default class Interaction extends React.Component {
```
[the remark this answers is not in the document]
Comment (paulgain) [reply src/activity-feed/activity-feed-cards/Omis.jsx]: Done.

File: src/activity-feed/activity-feed-cards/Omis.jsx (hunk 2 above)
Comment (paulgain) [reply src/activity-feed/activity-feed-cards/Omis.jsx]: 👍

File: src/activity-feed/activity-feed-cards/Omis.jsx (hunk 3 above)
Comment (paulgain) [reply src/activity-feed/activity-feed-cards/Omis.jsx]: This is an inconsistency with the designs.

File: fixtures/activity_feed/index.js (hunk 4 above)
Comment (paulgain) [reply fixtures/activity_feed/index.js]: 👍

**Pull request closed without merging**